packages/node_modules/@node-red/editor-client/src/js/ui/editors/code-editors/monaco.js

```diff
@@ -563,7 +563,7 @@ RED.editor.codeEditor.monaco = (function() {
                     createMonacoCompletionItem("node.send", 'node.send(${1:msg});','async send a msg to the next node',range),
                     createMonacoCompletionItem("node.send (multiple)", 'var ${1:msg1} = {payload:${2:1}};\nvar ${3:msg2} = {payload:${4:2}};\nnode.send([[${1:msg1}, ${3:msg2}]]);','send 1 or more messages out of 1 output',range),
                     createMonacoCompletionItem("node.send (multiple outputs)", 'var ${1:msg1} = {payload:${2:1}};\nvar ${3:msg2} = {payload:${4:2}};\nnode.send([${1:msg1}, ${3:msg2}]);','send more than 1 message out of multiple outputs',range),
-                    createMonacoCompletionItem("node.status", 'node.status({fill:"${1:red}",shape:"${2:ring}",text:"${3:error}"});','Set the status icon and text underneath the function node',range),
+                    createMonacoCompletionItem("node.status", 'node.status({fill:"${1|red,green,yellow,blue,grey|}",shape:"${2|ring,dot|}",text:"${3:message}"});','Set the status icon and text underneath the function node',range),
```

Comment:
As suggested by Dave C - added "node.status snippet enums for fill and shape" 7d24e5b ...
![MqPjeAEFE0](https://user-images.githubusercontent.com/44235289/121644924-870d6900-ca8b-11eb-857a-73b2522d2c08.gif)